### DISCUSSION TITLE
own_capabilities extraData missing after update

### DISCUSSION THREAD

**Discussion by Brazol** [GetStream/stream-chat-flutter#1045] [Q&A] [answered]
I'm watching `channel.extraDataStream` and printing its content and it looks like after calling `channel.updateName` I get a new value in the stream (as expected) but without `own-capabilities`. Is this expected behaviour? 

And that is the state in which the channel is then cached and from now on when checking permissions there are none.

**Comment by Brazol** [comment]
I updated the dependency to use version 4.0.0-beta.1 and now it works fine when I use property `ownCapabilities` instead of extraData.

**Comment by xsahil03x** [comment, marked as answer]
Fixed in https://github.com/GetStream/stream-chat-flutter/pull/1049
